dictionary.py

```diff
@@ -88,22 +104,23 @@ def remove(self, reading):
 def write_index_header(stream):
     stream.write('<?xml version="1.0" encoding="UTF-8"?>\n')
     stream.write('<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">\n')
-    stream.write('<html xmlns:idx="www.mobipocket.com" xmlns:mbp="www.mobipocket.com" xmlns="http://www.w3.org/1999/xhtml">\n')
```

Comment:
I just noticed you forgot to close the html element!   No idea how this even worked since.

Anyway, we can't pretend this is XML any longer, as XML requires quotes on attributes.